Makefile

```diff
@@ -16,14 +16,11 @@ TEST_IMAGE=$(REPO)-testimages:$(VERSION)
 GOOS=linux
 GOARCH=amd64
 
-.PHONY: all dep build check clean test
+.PHONY: all  build check clean test
 
-all: check test build
+all: check build 
 
-dep:
-	dep ensure -v
-
-build: dep build-go build-image
+build: build-go build-image
```

Comment:
`make dep` is removed as we no longer rely on `dep` to pull our dependencies. But we perhaps need to add `go mod tidy` here. Else in a new environment how does the user know how to pull dependencies?

Reply:
We perhaps also need to set `export GO111MODULE=on`.... please check if its needed and if yes, create a new make target that will do all necessary steps for pulling dependencies using gomodules. This target should replace the earlier "dep" target everywhere in the Makefile.
https://github.com/golang/go/wiki/Modules#how-to-define-a-module

Reply:
we don't need to do make dep as while running go build dependencies will be pulled based on go.mod file

Reply:
We don't need export GO111MODULE=on as we are using go version 1.13 which by default will use the go.mod file if present this property value is needed for version below 1.13 that to when the project is inside the GOPATH to make use of go.mod file while downloading dependencies